Formula/m/movgrab.rb

```diff
@@ -32,7 +32,17 @@ class Movgrab < Formula
 
   # Backport fix for GCC linker library search order
   # Upstream ref: https://github.com/ColumPaget/Movgrab/commit/fab3c87bc44d6ce47f91ded430c3512ebcf7501b
-  patch :DATA
+  patch do
+    url "https://raw.githubusercontent.com/Homebrew/formula-patches/6e5fdfd05ce62383c7f3ac4b23ba31f5ffbac5b2/movgrab/linker.patch"
+    sha256 "e23330f110cb8ea2ed29ebc99180250fa5498d53706303b4d1878dc44aa483d3"
+  end
+
+  # build patch to fix pointer conversion issues
+  # upstream bug report, https://github.com/ColumPaget/Movgrab/issues/6
+  patch do
+    url "https://raw.githubusercontent.com/Homebrew/formula-patches/ba252015727b6f0fb362fec3edfb7c53a3f888c2/movgrab/pointer-conv.patch"
+    sha256 "9b5c0bb666d92c87966e610e3c2db9736371507b646359b5421f2a4fa7d68222"
+  end
```

Comment:
Where did this patch come from? I have doubts about its correctness.

Reply:
the patch comes from me, it is mostly fixing the incompatibility issue and pretty straightforward?

Reply:
if we have concern over this patch, I am also happy just to deprecate this formula (the formula has not been actively maintained over the past five years 🤷‍♂️ )

Reply:
Not sure it's pretty straightforward, but it does seem to fix a bug in the code. I think we can merge this...

Reply:
totally 👍